### PR TITLE
release-controller: Add generated deployments for RHEL 8.4 repos

### PR DESCRIPTION
**Pull request by travier**
Generated with:
./hack/generators/release-controllers/generate-release-controllers.py .

Fixes: https://github.com/openshift/release/pull/17620

### CODE REVIEW

File: clusters/build-clusters/common/release-controller/rpms-ocp-4.8.yaml [hunk 1]
```diff
@@ -488,6 +488,86 @@ spec:
...
+kind: Service
+metadata:
+  name: base-4-8-rhel84
+  namespace: ocp
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: base-4-8-rhel84
```
Comment (cgwalters) [inline clusters/build-clusters/common/release-controller/rpms-ocp-4.8.yaml:501]: Let's just remember to remove this by the time RHEL 84 rolls around!

**Review by cgwalters — APPROVED** [review]

**Comment by miabbott** [comment]
Looks sane to me

/approve
/assign @bbguimaraes

**Comment by jupierce** [comment]
/hold
hrrm.. This shouldn't pass CI checks without the corresponding change to the generator python checked in. @bradmwilliams can you take a look?
I'll leave this open for investigation and open another PR that doesn't alter the generated files.

**Comment by jupierce** [comment]
Found the issue in the PR check and I'll fix in a separate pull.
/unhold

**Pull request merged**